assets/src/components/selection/event-select/index.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { stringify } from 'querystringify';
+import moment from 'moment';
+import { isUndefined, pickBy, reduce, isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * WP dependencies
+ */
+const { Component } = wp.element;
+const { Placeholder, SelectControl, withAPIData } = wp.components;
+const { __ } = wp.i18n;
+
+const nowDateAndTime = moment();
```

Comment:
Should this maybe be moved up with the External dependencies?
Its current location almost makes it appear as though it is a WP dependency.

Reply:
ya good point, however the extra space I think delineates its not part of the block.

Reply:
ya that's fine and I knew you would say that, but initially I was just looking for some kind of comment to add !